hubspot_commerce.module

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * Hubspot_commerce.module.
+ */
```

Comment:
**CHANGE REQUEST**

The second line here should be descriptive of what the file provides. Something like "Hooks and functionality for the Commerce HubSpot module.